scripts/linters/js_ts_linter.py

```diff
@@ -220,6 +228,66 @@ def _compile_ts_file(self, filepath, dir_path):
             dir_path, os.path.basename(filepath).replace('.ts', '.js'))
         return compiled_js_filepath
 
+    def _check_any_type(self):
+        """Checks if the type of any variable is declared as any
+        in typescript files.
+        """
+
+        if self.verbose_mode_enabled:
+            python_utils.PRINT('Starting any type check')
+            python_utils.PRINT('----------------------------------------')
+
+        # pylint: disable=invalid-punctuation-used
```

Comment:
Why is this added?

Reply:
`:` is a invalid punctuation according to lint checks.

Reply:
@Hudda Could you look into this, please?

Reply:
@vojtechjelinek @nishantwrp I ran this code locally and I'm not seeing any error even after removing `# pylint: disable=invalid-punctuation-used`.

Reply:
@Hudda weird. It's working now. Removed that line.

Reply:
@nishantwrp This was because you didn't have '.' at the end of your comments.

Reply:
Ah, okay.